src/lib/dlt_user.c

```diff
@@ -414,6 +414,18 @@ DltReturnValue dlt_init(void)
     return DLT_RETURN_OK;
 }
 
+void dlt_get_appid(char * appid)
+{
+    if(appid != NULL)
+    {
+        strncpy(appid, dlt_user.appID, 4);
+    }
+    else
+    {
+        dlt_log(LOG_ERR, "Invalid parameter.\n");
```

Comment:
From #187 's use case, it's better to return error value instead of just printing error message.

Reply:
I think #187's user case care about the value of appid but not the return value. In case of error, appid will be NULL or empty. Isn't it?

Reply:
Actually, it depends what the caller set to the parameter since this function won't do any initialization...